src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -1345,13 +1345,14 @@ public static File inputStreamToFile(InputStream inputStream) throws IOException
             return null;
         }
         File file = File.createTempFile(UUID.randomUUID().toString(), UUID.randomUUID().toString());
-        OutputStream outputStream = new FileOutputStream(file);
+        try(OutputStream outputStream = new FileOutputStream(file)){
         int read = 0;
         byte[] bytes = new byte[1024];
         while ((read = inputStream.read(bytes)) != -1) {
             outputStream.write(bytes, 0, read);
         }
         return file;
+	}
```

Comment:
lines 161 and 162 have the same stream and channel issue - the input and output streams aren't closed.